docset/winserver2022-ps/failoverclusters/Add-ClusterSharedVolumeToAffinityRule.md

```diff
@@ -0,0 +1,199 @@
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [-PassThru] [<CommonParameters>]
+```
+
+### InputObject (cdxml)
+
+```
+Add-ClusterSharedVolumeToAffinityRule -InputObject <CimInstance[]> 
+ [-ClusterSharedVolumes] <String[]> [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [-PassThru]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet allows you to add cluster shared volumes (CSVs) to an existing affinity rule, using
+either the CSV name or object.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Add-ClusterSharedVolumeToAffinityRule -ClusterSharedVolumes MyVolume -Name MyRule -Cluster MyCluster
+```
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+The cmdlet immediately returns an object that represents the job and then displays the command
+prompt. You can continue to work in the session while the job completes. To manage the job, use the
+`*-Job` cmdlets. To get the job results, use the
+[Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet.
+
+For more information about Windows PowerShell background jobs, see
+[about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227967)
+or [Get-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ClusterSharedVolumes
+
+The cluster shared volumes to be added to the affinity rule.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+
+Specifies the input object that is used in a pipeline command.
+
+```yaml
+Type: CimInstance[]
+Parameter Sets: InputObject (cdxml)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Name
+
+The affinity rule in question to add cluster shared volumes to.
+
+```yaml
+Type: String[]
+Parameter Sets: Query (cdxml)
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -PassThru
+
+Returns an object representing the item with which you are working. By default, this cmdlet doesn't
```

Comment:
Instead of `the item with which you are working`, can this be more specific? What is the output object type? Is this the updated rule, or something else?